custom_components/tuya_local/devices/pgst_pw150_alarm.yaml

```diff
@@ -0,0 +1,195 @@
+name: Alarm system
+products:
+  - id: bxtloihrivkxalgn
+    name: PGST PW150 WiFi Home Alarm System
+primary_entity:
+  entity: alarm_control_panel
+  dps:
+    - id: 1
+      type: string
+      name: alarm_state
+      mapping:
+        - dps_val: disarmed
+          value: disarmed
+        - dps_val: arm
+          value: armed_away
+        - dps_val: home
+          value: armed_home
+        - dps_val: sos
+          value: triggered
+    - id: 1
+      type: string
+      name: trigger
+      mapping:
+        - dps_val: disarmed
+          value: false
+        - dps_val: arm
+          value: false
+        - dps_val: home
+          value: false
+        - dps_val: sos
+          value: true
+        - value_redirect: alarm_state
+    - id: 24
+      type: string
+      name: zone_attribute
+      optional: true
+    - id: 32
+      type: string
+      name: master_state
+      optional: true
+    - id: 36
+      type: string
+      name: sub_class
+      optional: true
+    - id: 37
+      type: string
+      name: sub_type
+      optional: true
+    - id: 38
+      type: string
+      name: sub_admin
+      optional: true
+    - id: 39
+      type: string
+      name: sub_state
+      optional: true
+      readonly: true
+    - id: 103
+      type: string
+      name: zone_number
+      optional: true
+      readonly: true
+    - id: 104
+      type: string
+      name: other_event
+      optional: true
+      readonly: true
+secondary_entities:
+  - entity: number
+    name: Exit delay
+    category: config
```

Comment:
Maybe add an icon, or `translation_key: timer`, same for siren duration and entry delay below
The timer translation will use the same standard timer icon, so to differentiate them, icons like "mdi:clock-start", "mdi:clock-end" can be used instead.